docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+bar
```

Comment:
@S0lll0s If you do not want to make bar uppercase write author name there. It is not the only package named “bar” in any case: I have `sys-apps/bar` from http://clpbar.sourceforge.net/ which is “Console Progress Bar” in main portage tree.